modules/RaptMediaNg/RaptMediaNg.json

```diff
@@ -0,0 +1,34 @@
+{
+	"mw.RaptMediaNg": {
+		"scripts": [
+			"resources/engine.min.js",
+			"resources/mw.RaptMediaNg.js"
+		],
+		"styles": "resources/RaptMediaNgPlugin.css",
```

Comment:
if (not sure) this is the same CSS as prev version - we can use it too.

Reply:
The CSS has changed slightly and so can't be de-duplicated.